src/ca/ubc/cs304/database/DatabaseConnectionHandler.java

```diff
@@ -44,7 +37,7 @@ public void close() {
 	}
 
 	public void addRequiredTables() {
-        ScriptRunner sr = new ScriptRunner(connection);
+	    // ScriptRunner sr = new ScriptRunner(connection);
```

Comment:
this needs un-commenting. We need the `ibatis-core-3.0.jar` library to access this.